members/members/AlexVCS.ts

```diff
@@ -42,5 +42,10 @@ export const AlexVCS: MemberObject = {
 		// { type: 'hashnode', username: 'yourUserName' },
 		// { type: 'website', url: 'https://virtualcoffee.io', title: 'Title of link' },
 	],
-	badges: ['Hacktoberfest2022'],
+	badges: ['Hacktoberfest2022'], ['Hacktoberfest2023'],
```

Comment:
Looks like we need to fix up the syntax here

```suggestion
	badges: ['Hacktoberfest2022' 'Hacktoberfest2023'],
```